Makefile

```diff
@@ -1,19 +1,11 @@
-SHELL := /usr/bin/env bash
-
-OPERATOR_DOCKERFILE = ./build/Dockerfile
-
-# Include shared Makefiles
-include project.mk
-include standard.mk
+include boilerplate/generated-includes.mk
 
-default: gobuild
-
-# Extend Makefile after here
+SHELL := /usr/bin/env bash
 
-# Build the docker image
-.PHONY: docker-build
-docker-build: build
+.PHONY: boilerplate-update
+boilerplate-update:
+	@boilerplate/update
 
-# Push the docker image
-.PHONY: docker-push
-docker-push: push
+# TODO: Remove below targets once prow config standardized
+.PHONY: gobuild
+gobuild: go-build
```

Comment:
Don't need to do this, since prow isn't actually using `gobuild` directly. Instead, fix the [callout in the Dockerfile](https://github.com/openshift/certman-operator/pull/170/files?file-filters%5B%5D=.go&file-filters%5B%5D=.mk&file-filters%5B%5D=No+extension#diff-5aecd73bbcb9b7410938673269f0d4e676dd18509277b0ff2f2787c9d3e22604R8).

Reply:
Thanks, I've updated this.

Reply:
:+1: